MC/bin/o2dpg_sim_workflow.py

```diff
@@ -1208,7 +1208,12 @@ def addQCPerTF(taskName, needs, readerCommand, configFilePath, objectsFile=''):
      addQCPerTF(taskName='trdDigitsQC',
                 needs=[TRDDigitask['name']],
                 readerCommand='o2-trd-trap-sim',
-                configFilePath='json://${O2DPG_ROOT}/MC/config/QC/json/trd-digits-task.json')
+                configFilePath='json://${O2DPG_ROOT}/MC/config/QC/json/trd-standalone-task.json')
+
+     addQCPerTF(taskName='trdTrackingQC',
+                needs=[TRDTRACKINGtask2['name']],
+                readerCommand='o2-global-track-cluster-reader --track-types "ITS-TPC-TRD" --cluster-types none',
```

Comment:
I would assume that the reader needs to be configured correctly to take into account the TPC-TRD matches.